driver/kubernetes/manifest/manifest.go

```diff
@@ -91,18 +91,18 @@ func NewDeployment(opt *DeploymentOpt) (d *appsv1.Deployment, c []*corev1.Config
 		labels[k] = v
 	}
 
-	d = &appsv1.Deployment{
+	s = &appsv1.StatefulSet{
 		TypeMeta: metav1.TypeMeta{
 			APIVersion: appsv1.SchemeGroupVersion.String(),
-			Kind:       "Deployment",
+			Kind:       "StatefulSet",
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace:   opt.Namespace,
 			Name:        opt.Name,
 			Labels:      labels,
 			Annotations: annotations,
 		},
-		Spec: appsv1.DeploymentSpec{
+		Spec: appsv1.StatefulSetSpec{
```

Comment:
`podManagementPolicy` should be set to `Parallel`

Reply:
Also please make sure to measure the startup time of the pods with a good number of replicas (around 5-10?), and there is no significant delay compared to Deployment